### ISSUE TITLE
Sync monitor position to the spaces group position in the menu bar

### ISSUE THREAD

**Issue by program-om** [Jaysce/Spaceman#33]
I have two external displays
<img width="569" alt="Screenshot 2024-04-16 at 6 11 03 PM" src="https://github.com/Jaysce/Spaceman/assets/18104843/128434a5-3777-432e-9328-2505bf6de621">

The display on the right side has 4 spaces.
The display on the left side has 3 spaces.

However, the order of display group in the menu bar is swapped
<img width="214" alt="Screenshot 2024-04-16 at 6 12 31 PM" src="https://github.com/Jaysce/Spaceman/assets/18104843/25e90bf4-ffec-48ce-a18d-90eb102842ec">
I would expect the four spaces group to be on the right and the 3 spaces group to be on the left

**Comment by john8329** [comment]
I agree that'd be useful, not sure though if there's a way for the app to display a different version of its widget on each menu bar.

**Comment by mgumz** [comment]
@john8329 nah, the idea is not different widgets per menu-bar.

@program-om wants to have `[5,6,7] [1,2,3,4]` because that is how the order of his/her monitors is: the left monitor hosts [5,6,7] and the right monitor hosts [1,2,3,4]

**Comment by ruittenb** [comment]
This problem has been addressed by @mike-jl and I have merged it in my recent fork, which can be found at https://github.com/ruittenb/Spaceman

**Comment by mgumz** [comment]
#35 is tackling the issue, @john8329 please try it

**Comment by roelvangils** [comment]
> This problem has been addressed by @mike-jl and I have merged it in my recent fork, which can be found at [ruittenb/Spaceman](https://github.com/ruittenb/Spaceman)

Wow, that's amazing! That's exactly the reason I came to the issues section of this repository: to find out if anyone else has requested this. I'm going to try out your fork right away.
